### PR TITLE
Remove hardcoded plugin name for getting MAX_TEXTURES in Graphics.ts

### PR DESCRIPTION
**Pull request by SergeiSomin**
##### Description of change
Remove hardcoded plugin name for getting MAX_TEXTURES values, because the plugin name should be used from pluginName variable.

##### Pre-Merge Checklist
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

File: packages/graphics/src/Graphics.ts [hunk 1]
```diff
@@ -1024,7 +1024,7 @@ export class Graphics extends Container
             // but may be more than one plugins for graphics
             if (!DEFAULT_SHADERS[pluginName])
             {
-                const MAX_TEXTURES = renderer.plugins.batch.MAX_TEXTURES;
+                const MAX_TEXTURES = renderer.plugins[pluginName].MAX_TEXTURES;
```
Comment (bigtimebuddy) [inline packages/graphics/src/Graphics.ts:1027]: Small style improvement:
```suggestion
                const { MAX_TEXTURES } = renderer.plugins[pluginName];
```

**Review by bigtimebuddy — COMMENTED** [review]
This seems reasonable to me. Can you tell me how you bumped into this error? Would it be possible to create a test that captures this?

**Comment by SergeiSomin** [comment]
hi, thanks, i just tried to understand how pixi rendering works and saw it
I think to test it in theory we could change the renderer plugin to some mocked one and then we could check the amount of samplers in shader if it is the same as MAX_TEXTURES in mocked renderer

**Review by bigtimebuddy — APPROVED** [review]
This was introduced here, I think it was just an oversight #6876

**Comment by bigtimebuddy** [comment]
Thank you @SergeiSomin!

**Pull request merged**